src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -2277,7 +2280,8 @@ void CEndlessUsbToolDlg::UpdateFileEntries(bool shouldInit)
             CString displayName, personality, version, date;
             bool isInstallerImage = false;
             if (!ParseImgFileName(trueName, personality, version, date, isInstallerImage)) continue;
-            ULONGLONG extractedSize = GetExtractedSize(currentFilePath, isInstallerImage);
+            CompressionType compressionType;
+            ULONGLONG extractedSize = GetExtractedSize(currentFilePath, isInstallerImage, compressionType);
```

Comment:
I agree. It could return `std::tuple<ULONGLONG, CompressionType>` but unless you have destructuring assignment (?) that would be syntactically noisy. One for later, I think…